homeassistant/components/homekit/type_cameras.py

```diff
@@ -171,19 +174,31 @@ async def start_stream(self, session_info, stream_config):
             return False
         if "-i " not in input_source:
             input_source = "-i " + input_source
+        video_profile = ""
+        if self.config[CONF_VIDEO_CODEC] != "copy":
+            video_profile = (
+                "-profile:v "
+                + VIDEO_PROFILE_NAMES[
+                    int.from_bytes(stream_config["v_profile_id"], byteorder="big")
+                ]
+                + " "
+            )
+        audio_application = ""
+        if self.config[CONF_AUDIO_CODEC] == "libopus":
+            audio_application = "-application lowdelay "
         output_vars = stream_config.copy()
         output_vars.update(
             {
-                "v_profile": VIDEO_PROFILE_NAMES[
-                    int.from_bytes(stream_config["v_profile_id"], byteorder="big")
-                ],
+                "v_profile": video_profile,
                 "v_bufsize": stream_config["v_max_bitrate"] * 2,
```

Comment:
```suggestion
                "v_bufsize": stream_config["v_max_bitrate"] * 4,
```

This fixed the dropouts for me

Reply:
fixed.